rollingpin/frontends.py

```diff
@@ -160,6 +161,28 @@ def on_deploy_end(self):
         elapsed = time.time() - self.start_time
         print "*** elapsed time: %d seconds" % elapsed
 
+        # TODO: Make this smarter.  It should only show anomolous
+        # stuff.  Or maybe color that stuff differently.
+        #
+        # TODO: Check if component report was even requested before going
+        # through them all?
+        report = collections.defaultdict(lambda: collections.Counter())
+        for host, results in self.host_results.iteritems():
+            # Messed up hosts won't have results
+            if 'results' not in results:
+                continue
+            for command, output in results['results']:
+                if command[0] != 'component_report':
```

Comment:
Can the command be named with a hyphen instead of underscore for consistency with the other commands please?